### PR TITLE
V1.36.1-1 stage

### PR DESCRIPTION
**Pull request by yunhee-l**
### Summary
ecs init release 1.36.0

Cache Agent version 1.36.1
Fixing potential file descriptor leak from context logger

### Testing

New tests cover the changes: 

### Licensing

This contribution is under the terms of the Apache 2.0 License:

### CODE REVIEW

File: packaging/amazon-linux-ami/ecs-init.spec [hunk 1]
```diff
@@ -255,6 +255,9 @@ fi
 %endif
 
 %changelog
+* Wed Jan 15 2020 Yunhee Lee <yhlee@amazon.com> - 1.36.1-1
+- Cache Agent version 1.36.1
+- Fix potential file descriptor leak from context logger
```
Comment (sharanyad) [inline packaging/amazon-linux-ami/ecs-init.spec]: is this needed in init? since the changes are in agent

Comment (fenxiong) [inline packaging/amazon-linux-ami/ecs-init.spec]: this is not needed imo since it's not a change in ecs init itself

Comment (yunhee-l) [reply packaging/amazon-linux-ami/ecs-init.spec]: It's in the release instructions, and looks like previous releases have updated also.

Comment (yunhee-l) [reply packaging/amazon-linux-ami/ecs-init.spec]: I'll remove agent fix details.

File: packaging/ubuntu-trusty/debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+amazon-ecs-init (1.36.1-1) trusty; urgency=medium
+
+  * Cache Agent version 1.36.1
+  * Fix potential file descriptor leak from context logger
```
Comment (sharanyad) [inline packaging/ubuntu-trusty/debian/changelog]: missed removing this

File: packaging/suse/amazon-ecs.service [hunk 3]
```diff
@@ -10,7 +10,8 @@ Requires=network.target
 Type=simple
 ExecStartPre=/usr/sbin/amazon-ecs-init pre-start
 ExecStart=/usr/sbin/amazon-ecs-init start
-ExecStop=/usr/sbin/amazon-ecs-init pre-stop
+ExecStop=/usr/sbin/amazon-ecs-init stop
```
Comment (sharanyad) [inline packaging/suse/amazon-ecs.service]: are we expecting this change to be in here?

Comment (petderek) [reply packaging/suse/amazon-ecs.service]: cc @fierlion is this safe to go out?

File: packaging/ubuntu-trusty/debian/changelog (hunk 2 above)
Comment (yunhee-l) [reply packaging/ubuntu-trusty/debian/changelog]: fixed!

File: packaging/suse/amazon-ecs.service (hunk 3 above)
Comment (sharanyad) [reply packaging/suse/amazon-ecs.service]: if yes, does this need changelog?

Comment (yunhee-l) [reply packaging/suse/amazon-ecs.service]: yes this is a change unrelated to fd leak we decided to keep for this releaese.

Comment (yunhee-l) [reply packaging/suse/amazon-ecs.service]: https://github.com/aws/amazon-ecs-init/issues/162 we'll address it with this issue

**Pull request merged**